nms/app/packages/magmalte/app/views/subscriber/SubscriberAddDialog.js

```diff
@@ -518,14 +519,17 @@ function AddSubscriberDetails(props: DialogProps) {
             sub_profile: subscriber.dataPlan,
           },
         });
-        enqueueSnackbar('Subscriber(s) saved successfully', {
-          variant: 'success',
-        });
+        successCount++;
       } catch (e) {
         const errMsg = e.response?.data?.message ?? e.message ?? e;
         setError('error saving ' + subscriber.imsi + ' : ' + errMsg);
         return;
       }
+      if (successCount % 50 === 0 || successCount === subscribers.length) {
+        enqueueSnackbar(`${successCount} Subscriber(s) saved successfully`, {
```

Comment:
This is the nittiest nit I've ever made but I'm always mad when websites don't implement simple checks to write plural vs. singular correctly. When we get back to the office I'll give you a cookie if you make this say "subscriber" or "subscribers" based on the number.